final_prep/README.md

```diff
@@ -32,14 +32,15 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- ![image](https://user-images.githubusercontent.com/78521446/110194253-e60c7380-7df4-11eb-8928-013b60b7bd7f.png)
```

Comment:
Great work setting up a calendar. Don't be afraid to change it up and experiment with what works for you